go.mod

```diff
@@ -11,6 +11,7 @@ require (
 	github.com/bradfitz/gomemcache v0.0.0-20180710155616-bc664df96737
 	github.com/codahale/hdrhistogram v0.0.0-20161010025455-3a0bb77429bd // indirect
 	github.com/codegangsta/cli v1.20.0
+	github.com/davecgh/go-spew v1.1.1
```

Comment:
Doesn't feel like this changte belongs here?

Reply:
We already use `spew` dep in production build - https://github.com/grafana/grafana/blob/0833f26b4337089af99b1b731d7e19588333e061/pkg/services/ldap/ldap.go#L182 and https://github.com/grafana/grafana/blob/0833f26b4337089af99b1b731d7e19588333e061/go.sum#L27. But we are missing it `go.mod`.

In other words that's the product of running `go mod tidy`